README.md

```diff
@@ -7,3 +7,24 @@
 <h2>Pre-Proudction. Currently only includes stroke.</h2>
 <p>After installing, you'll find a "stroke" filter for your sources.  Select this filter, and play with the settings.  Enjoy.</p>
 <p><i>Note- a much better readme is coming soon.</i></p>
+
+### For Ubuntu
+
+#### With GUI
+
+- Download the Ubuntu .tar.gz for your Ubuntu version (currently 20.04 and 22.04 are supported)
+- Extract the .tar.gz to a handy location.
+- Using your file manager, navigate to `~/.config/obs-studio/plugins`
```

Comment:
On my system, the _plugins_ directory did not exist.  I had to manually create it and then follow the rest of the instructions.  Please consider updating these docs.

Reply:
Updated instructions. The wording may not be great.